server/pulp/server/managers/repo/unit_association_query.py

```diff
@@ -23,6 +23,7 @@
 
 _VALID_DIRECTIONS = (SORT_ASCENDING, SORT_DESCENDING)
 
+UNITS_CHUNK_SIZE = 100000
```

Comment:
i would rather name this variable units_batch_size, chunk can be confusing

Reply:
Replaced UNITS_CHUNK_SIZE with UNITS_BATCH_SIZE.